examples/step-37/doc/results.dox

```diff
@@ -402,6 +402,32 @@ processors in MPI, which limits scalability by a factor of around two to five.
 
 <h3> Possibilities for extensions</h3>
 
+<h4> Kelly error estimator </h4>
+
+As mentioned above the code is ready for the locally adaptive h-refinement.
+For the Poisson equation one can employ the Kelly error indicator,
+implemented in the KellyErrorEstimator class. However one needs to be carefull
+with the ghost indices of parallel vectors.
+In order to evaluate the jump terms in the error indicator, each MPI process
+needs to know locally relevant DoFs. However for performance reasons the
+MatrixFree::initialize_dof_vector() function initializes the vector only with
+some locally relevant DoFs. Consequently the solution vector as-is is
```

Comment:
Maybe expand this a bit and move the part with the performance reasons to the end:
The ghost indices made available in the vector are a tight set of only those indices that are touched in the cell integrals (including constraint resolution). This choice has performance reasons, because sending all locally relevant degrees of freedom would be too expensive compared to the matrix-vector product.

(If we want to be more specific, we could add that the ghost exchange typically amounts to 5-35% of the total cost of the matrix-vector product; the locally relevant section would increase this portion especially for high-order finite elements where there are fewer elements and the volume-to-surface ratio is less beneficial.)